src/pilot/deploy-overcloud.py

```diff
@@ -664,6 +664,8 @@ def main():
                             help="Indicates if the deploy-overcloud script "
                                  "should be run in debug mode")
         args = parser.parse_args()
+
+        LoggingHelper.configure_logging()
```

Comment:
This method requires arguments and tried adding this but it didn't work for me.
So I changed 
logging.basicConfig(level=logging.INFO)
to 
logging.basicConfig(level=logging.CRITICAL) 
in deploy-overcloud.py
and fixed it for now.

Do we require INFO messages as well ?